tests/providers/docker/operators/test_docker_swarm.py

```diff
@@ -77,6 +77,7 @@ def _client_service_logs_effect():
             image='ubuntu:latest',
             command='env',
             user='unittest',
+            mounts=[],
```

Comment:
IIUC this test would pass even if there were no changes made to `DockerSwarmOperator` in this PR (since we're checking the default value of `DockerOperator` right now).
I think it is worth passing some value to the operator & checking it gets passed here as expected.

Reply:
I’ve modified the test case to ensure the variable is passed.